### PR TITLE
full_figure_for_development

### PR DESCRIPTION
**Pull request by nicolaskruchten**
Addresses the non-error portion of #1967

@jonmmease I'm not sure how to best test this or if such a trivial function is worth a test... ?

### CODE REVIEW

File: packages/python/plotly/plotly/io/_kaleido.py [hunk 1]
```diff
@@ -260,4 +261,58 @@ def write_image(
...
+    -------
+    plotly.graph_objects.Figure or dict
+        The full figure
+    """
+
+    # Raise informative error message if Kaleido is not installed
+    if scope is None:
+        raise ValueError(
+            """
+Image export using the "kaleido" engine requires the kaleido package,
```
Comment (jonmmease) [inline packages/python/plotly/plotly/io/_kaleido.py:294]: Should probably refer to full_figure_for_development instead of Image export here

Comment (nicolaskruchten) [reply packages/python/plotly/plotly/io/_kaleido.py]: thanks! done

**Comment by nicolaskruchten** [comment]
OK added a test and some docs!

**Comment by nicolaskruchten** [comment]
OK, all done! I had to add `.decode("utf-8")` to get bytes into strings for the Python 3.5 optional tests to pass ... normal? that'll work on 2.7 also right?

**Comment by nicolaskruchten** [comment]
also I added a doc page... feedback welcome :)  @emmanuelle if you want to take a look that'd be cool too :)

**Comment by jonmmease** [comment]
> OK, all done! I had to add .decode("utf-8") to get bytes into strings for the Python 3.5 optional tests to pass ... normal? that'll work on 2.7 also right?

Not sure about the Python 3.5 thing specifically, but decoding explicitly to utf-8 is the right thing to do.

**Pull request merged**